lib/js/dom.nim

```diff
@@ -1318,6 +1345,29 @@ proc offsetLeft*(e: Node): int {.importcpp: "#.offsetLeft", nodecl.}
 since (1, 3):
   func newDomParser*(): DOMParser {.importcpp: "new DOMParser()".}
     ## DOM Parser constructor.
-
   func parseFromString*(this: DOMParser; str: cstring; mimeType: cstring): Document {.importcpp.}
     ## Parse from string to `Document`.
+
+  proc newDomException*(): DomException {.importcpp: "new DomException()", constructor.}
```

Comment:
```suggestion
  proc newDomException*(): DomException {.importcpp: "(new DomException())", constructor.}
```

Reply:
I don't think you right, wrapping in the parenthesis is not required, have a look how all other objects in this module are implemented